src/index.js

```diff
@@ -4,30 +4,32 @@ import displayList from './displayList.js';
 import addScore from './addScore.js';
 
 const scoresList = document.querySelector('#scoresList');
-const scoresName = document.querySelector('#name');
+const scoresUser = document.querySelector('#user');
 const scoresNumber = document.querySelector('#score');
 const submitBtn = document.querySelector('#submitBtn');
+const refreshData = document.querySelector('#refreshData');
 
-const scores = [
-  { name: 'Vikita', score: 30 },
-  { name: 'Vikita', score: 30 },
-  { name: 'Vikita', score: 30 },
-  { name: 'Vikita', score: 30 },
-];
-
-displayList(scoresList, scores);
+refreshData.addEventListener('click', () => {
+  scoresList.innerHTML = '';
+  displayList(scoresList);
+});
```

Comment:
[ OPTIONAL]

- If I am clicking the refresh button more than one time the same is coming again. Suppose I clicked three times on refresh, I'm getting three times the same data. I have attached the screenshot below.

![Screenshot from 2021-09-01 18-25-38](https://user-images.githubusercontent.com/22434660/131675650-3daec59c-ed91-49d0-bd2a-4f66fc22f557.png)